### PR TITLE
[WIP]DOCTEAM-1336: Security Guide-style and spell checks

### PR DESCRIPTION
**Pull request by Amrita42**
### PR creator: Description

The scope is to remove fluff , usage of a words such as:
usually
Note that this
will
very
as long as >>if
Excessive and necessary usage of **only**
however
filesystem >> file system  (as per style guide)

Check for dead links and style issues  

### PR creator: Are there any relevant issues/feature requests?

[DOCTEAM-1336](https://jira.suse.com/browse/DOCTEAM-1336)

### PR creator: Which product versions do the changes apply to?

When opening a PR, check all versions of the documentation that your PR applies to.

- SLE 15/openSUSE Leap 15.x
  - [x] SLE 15 next/openSUSE Leap next *(current `main`, no backport necessary)*
  - [ ] SLE 15 SP5/openSUSE Leap 15.5
  - [ ] SLE 15 SP4/openSUSE Leap 15.4
  - [ ] SLE 15 SP3/openSUSE Leap 15.3
  - [ ] SLE 15 SP2/openSUSE Leap 15.2
  - [ ] SLE 15 SP1
- SLE 12
  - [ ] SLE 12 SP5
  - [ ] SLE 12 SP4

### PR reviewer only: Have all backports been applied?

The doc team member merging your PR will take care of backporting to older documents.
When opening a PR, do *not* set the following check box.

- [ ] all necessary backports are done

### CODE REVIEW

**Comment by Amrita42** [comment]
@dariavladykina please review , not urgent. Thanks

File: xml/security_firewall.xml [hunk 1]
```diff
@@ -723,7 +723,7 @@ dhcp
    <note>
     <title>About NFSv4</title>
     <para>
-     The newer NFSv4 only requires the single well known
+     The newer NFSv4 requires the single well known
```
Comment (dariavladykina) [inline xml/security_firewall.xml:726]:
```suggestion
     The newer NFSv4 requires the single well-known
```

File: xml/security_firewall.xml [hunk 2]
```diff
@@ -750,7 +750,7 @@ dhcp
      at your discretion but should not clash with any well known port numbers
```
Comment (dariavladykina) [inline xml/security_firewall.xml:750]:
```suggestion
     at your discretion but should not clash with any well-known port numbers
```

File: xml/security_firewall.xml [hunk 3]
```diff
@@ -872,10 +872,10 @@ dhcp
     </table>
 
     <para>
-     You will need to restart any related services that are affected by these
+     You need to restart any related services that are affected by these
      static port configurations for the changes to take effect. You can see
      the currently assigned rpcbind ports by using the command
-     <command>rpcinfo -p</command>. On success only the statically configured
+     <command>rpcinfo -p</command>. On success the statically configured
```
Comment (dariavladykina) [inline xml/security_firewall.xml:878]:
```suggestion
     <command>rpcinfo -p</command>. On success, the statically configured
```

File: xml/security_preface.xml [hunk 4]
```diff
@@ -532,7 +532,7 @@
    </listitem>
    <listitem>
     <para>
-     Use encryption where possible, for example for hard disks of mobile
+     Use encryption where possible, for example for hard disks of mobile.
```
Comment (dariavladykina) [inline xml/security_preface.xml:535]:
```suggestion
     Use encryption where possible, for example, for hard disks of mobile.
```

File: xml/security_preface.xml [hunk 5]
```diff
@@ -516,7 +516,7 @@
    </listitem>
    <listitem>
     <para>
-     Check your log files, for example with logwatch.
+     Check your log files, for example with Logwatch.
```
Comment (dariavladykina) [inline xml/security_preface.xml:519]:
```suggestion
     Check your log files, for example, with Logwatch.
```

**Review by dariavladykina — COMMENTED** [review]
Hi, please see some minor suggestions here. Thanks!

File: xml/apparmor_start.xml [hunk 6]
```diff
@@ -259,7 +259,7 @@
    There are two ways of managing profiles. One is to use the graphical
    front-end provided by the &yast; &aa; modules and the other is to
    use the command line tools provided by the &aa; suite itself. The main
```
[the remark this answers is not in the document]
Comment (Amrita42) [reply xml/apparmor_start.xml:261]: @dariavladykina as per the sytle guide , is it not correct as is , command line tools , as a noun.

[the remark this answers is not in the document]
Comment (dariavladykina) [reply xml/apparmor_start.xml:261]: It has the role of an adjective here, so the hyphen is needed.

**Pull request merged**